addons/web_editor/static/src/js/wysiwyg/widgets/link.js

```diff
@@ -87,7 +87,7 @@ const Link = Widget.extend({
                 $node = $node.parent();
             }
             const linkNode = this.$link[0] || this.data.range.cloneContents();
-            const linkText = linkNode.textContent;
+            const linkText = linkNode.innerText.replaceAll("\u200B", "");
```

Comment:
Earlier I said "this code seems to be the same in 16.0 blablabla", you retargeted to 16.0 etc. But it is actually the same in 15.0 and the commit you mentioned was made in 15.0 -> always check when a code was introduced and fix that over there please :pray: This should thus be retargeted to 15.0.